### ISSUE TITLE
Speed up docs CI/DC

### ISSUE THREAD

**Issue by simbilod** [gdsfactory/gplugins#161]
**Is your feature request related to a problem? Please describe.**
Docs build slow.

**Describe the solution you'd like**
The plugins are already tested separately. It would be great if the docs were also built on a plugin-per-plugin basis, since typically contributions target one specific plugin at a time.

**Comment by joamatab** [comment]
yes, we can run them with papermill and then jupyterbook just combines them, 

the main tricky part is to avoid the execution that do not work :) or catch any errors
